lib/ash/error/error.ex

```diff
@@ -106,8 +133,17 @@ defmodule Ash.Error do
   defp header(:framework), do: "Framework Error"
   defp header(:unknown), do: "Unknown Error"
 
+  @doc """
+  Returns whether or not a term is an Ash.Error type.
+  """
+  @spec ash_error?(term()) :: boolean()
   def ash_error?(value), do: splode_error?(value, __MODULE__)
 
+  @doc """
+  Sets the main path of the errors on the containing struct.
```

Comment:
This function prepends the provided path to any existing path on the errors, may be good to express that in the docs.